src/steps/users/contact-field-equals.ts

```diff
@@ -29,6 +29,7 @@ export class ContactFieldEquals extends BaseStep implements StepInterface {
   {
     field: 'operator',
     type: FieldDefinition.Type.STRING,
+    optionality: FieldDefinition.Optionality.OPTIONAL,
```

Comment:
- [ ] Recommend maintaining the `REQUIRED` aspect here.  The only reason we made this optional in other Cogs was to maintain backward compatibility for existing steps.